CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ We welcome contributions from anyone, and are grateful for even the smallest of
 
 ## Got a Question or Problem?
 
-For quick help, join the [Request Hub Slack](https://request-slack.herokuapp.com/) and ask your question there.
+For quick help, join the [Request Discord]( https://request.network/discord/) and ask your question there.
```

Comment:
Please remove the space after the first parenthesis `( https://request.network/discord/)`. This is what prettier is complaining about.